src/test/java/blackjack/domain/machine/ScoreTest.java

```diff
@@ -0,0 +1,65 @@
+package blackjack.domain.machine;
+
+import static blackjack.domain.card.Fixtures.SPADE_ACE;
+import static blackjack.domain.card.Fixtures.SPADE_EIGHT;
+import static blackjack.domain.card.Fixtures.SPADE_FOUR;
+import static blackjack.domain.card.Fixtures.SPADE_JACK;
+import static blackjack.domain.card.Fixtures.SPADE_TWO;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import blackjack.domain.card.Card;
+import blackjack.domain.card.Rank;
+import blackjack.domain.card.Suit;
+import java.util.Set;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class ScoreTest {
+
+    @Test
+    @DisplayName("스코어 클래스 생성 확인")
+    void createScore() {
+        Score score = new Score(Set.of(SPADE_ACE, SPADE_JACK));
```

Comment:
```suggestion
        Score score = new Score(SPADE_ACE, SPADE_JACK);
```

조금 더 사용성이 올라가지 않을까..